assets/home/tasks-section/tasks.scss

```diff
@@ -59,4 +61,16 @@
 		height: 25px;
 		justify-self: end;
 	}
+
+	&__external-icon {
+		width: 20px;
+		height: 20px;
+		margin-left: 5px;
+		color: currentColor;
```

Comment:
In order to not need to fix it for every new icon, I created https://github.com/Automattic/sensei/pull/7564/files#diff-01051aeda000eb6fe8ff5d366e6ccb554ce112970b13b9efe3539f9bc971dc06R63-R65 that we can apply. I didn't make it so generic in terms of `svg` selector because it doesn't work well for any icon. But it should always work well for `@wordpress/icons` icons because it's used in the editor.

WDYT about this approach? If you agree, do you think we could also use it here? If you agree, maybe copying is enough until we merge everything because it's part of a bigger commit for a cherry-pick.

Reply:
Yap makes Sense, I've updated it here https://github.com/Automattic/sensei/pull/7562/commits/8344632d3d9c4008a71282cfda49e91e2192dc0e . Just kept a bit of the style that is kinda specific to my icon